io_export_cryblend/export.py

```diff
@@ -745,20 +745,30 @@ def __process_bone_weights(self, object_, armature, skin_node):
         vw = ""
         vertex_groups_lengths = ""
         vertex_count = 0
+        vertex_group_count = 0

```

Comment:
You are right, c sindrome I think. I will delete later, I'm afraid that merging with a new commit github removes all the comments in the current commit.